src/Compilers/Core/Portable/StrongName/StrongNameFileSystem.cs

```diff
@@ -9,6 +9,13 @@ namespace Microsoft.CodeAnalysis
 {
     internal class StrongNameFileSystem
     {
+        private static StrongNameFileSystem s_StrongNameFileSystemInstance = new StrongNameFileSystem();
+        protected StrongNameFileSystem()
+        {
+        }
+
+        internal static StrongNameFileSystem GetInstance() => s_StrongNameFileSystemInstance;
```

Comment:
Consider making the field `internal` and removing the accessor method.

Reply:
Done.